requirements.txt

```diff
@@ -12,4 +12,4 @@ streamlit==1.7.0
 scipy>=1.7.3
 
 # Style
-black==21.12b0
```

Comment:
Either we do not fix the version, i.e. just write `black` here, or we update the workflow to be version `22.3.0`.